modules/nw-modifying-operator-install-config.adoc

```diff
@@ -22,15 +22,10 @@ ifeval::["{context}" == "installing-vmc-network-customizations-user-infra"]
 endif::[]
 
 [id="modifying-nwoperator-config-startup_{context}"]
-= Modifying advanced network configuration parameters
+= Specifying advanced network configuration
 
-You can modify the advanced network configuration parameters only before you
-install the cluster. Advanced configuration customization lets you integrate
-your cluster into your existing network environment by specifying an MTU or
-VXLAN port, by allowing customization of
-link:https://kubernetes.io/docs/reference/command-line-tools-reference/kube-proxy/[kube-proxy]
-settings, and by specifying a different `mode` for the `openshiftSDNConfig`
-parameter.
+You can specify advanced network configuration only before you install the cluster.
+Advanced configuration customization lets you integrate your cluster into your existing network environment by specifying additional configuration for your cluster network provider that is possible only during cluster installation.
```

Comment:
To avoid "let":

"You can use advanced configuration customization to integrate..."

Reply:
Is "that is possible only during cluster installation" redundant given the previous sentence? Though I can appreciate trying to drive it home.

Reply:
@adellape, true, I'll use less words.